test/WebJobs.Script.Tests.Integration/Host/StandbyManager/StandbyManagerE2ETests_Windows.cs

```diff
@@ -168,6 +173,105 @@ await TestHelpers.Await(() =>
             Assert.NotSame(GetCachedTimeZoneInfo(), _originalTimeZoneInfoCache);
         }
 
+        [Fact]
+        public async Task StandbyModeE2E_Dotnet_HostIdValidator_DoesNotRunInPlaceholderMode()
```

Comment:
This test reproduces the issue 100% of the time. I couldn't get a reliable repro since a race condition is involved, so as the comments below state, I resorted to forcing the condition via a direct blob write.